CreeDictionary/CreeDictionary/templates/CreeDictionary/paradigm.html

```diff
@@ -1,10 +1,26 @@
 {% load static %}
 {% load inflection_extras %}
-{# should be placed inside <main> #}
+{% comment %}
+
+This should be placed inside <main>
+
+{% endcomment %}
+
+{% spaceless %}
+{# This is not in the DOM by default, but JS places in the <h1 id="head"> #}
+<template id="template:play-button">
+    <button class="definition-title__play-button" data-cy="play-recording">
+        <svg aria-label="Play recording" class="definition-title__play-icon">
+            <use xlink:href="#fa-volume-up-solid" />
+        </svg>
+    </button>
+</template>
+{% endspaceless %}
```

Comment:
why spaceless though, just curious

Reply:
So that there is no space between XML tags! Obviously, spaces when coding make it more readable, but those spaces are just wasted bytes when they're transfered to the user, so just get rid of them!